bases/shared/entrypoint.sh

```diff
@@ -46,7 +46,7 @@ echo "/state/cores/core.%e.%p.%h.%t" > /proc/sys/kernel/core_pattern
 ln -sf "$SLOGFILE" /state/slogfile_current.json
 
 
-export MAX_VATS=70
+export MAX_VATS=150
```

Comment:
I am concerned with this hardcoded limit.  Maybe it is time to try using `otelcol-contrib` process monitoring again?

Reply:
I spent a of lot of time last time trying to figure out `otelcol-contrib` for the process.  Will try again sometime later.